scripts/close-milestone

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+set -x -e -o pipefail -u
+
+milestone_title=$1
+
+milestones_json_array=$(gh api -H "Accept: application/vnd.github+json" /repos/VirtusLab/git-machete-intellij-plugin/milestones)
+
+desired_milestone=$(echo "$milestones_json_array" | jq -c --arg TITLE $milestone_title '.[] | select(.title | contains($TITLE))')
+
+number=$(echo "$desired_milestone" | jq '.number')
+
+#open issues and PRs are bundled together in the same field
+open_issues=$(echo "$desired_milestone" | jq '.open_issues')
```

Comment:
Oh so `open_issues` are a **number** field (count of the issues/PRs), not the **list** of issues/PRs right?

Reply:
right.